test/routes/src/server/routes/t_github.js

```diff
@@ -34,24 +40,26 @@ describe('The GitHub API endpoint', () => {
         nock.cleanAll();
       });
 
-      it('should successfully return', (done) => {
-        supertest(app)
-          .get('/github/snapcraft-yaml/anowner/aname')
-          .expect(200, done);
+      it('should satisfy all request mocks', async () => {
```

Comment:
I wonder if moving this into it's own test isn't detrimental to debugging failed tests? Knowing with confidence that the mock was satisfied in a test is a good thing, and even though these tests hit the same endpoint without messing around perhaps this isn't a good pattern to establish? Maybe move to afterEach?